_data/users.yml

```diff
@@ -82,6 +82,20 @@
     data-driven insights.
   urltext:
   url: https://datappeal.io/
+- name: datasapiens
+  anchor: datasapiens
+  logo: /assets/images/logos/datasapiens.png
+  logosmall: /assets/images/logos/datasapiens-small.png
+  testimonial: |
+    At datasapiens, we help your company to become a data-driven organization.
```

Comment:
I almost suggested to capitalize datasapiens then looked at the website and seems like it is intentionally lower case. :)